docs/tutorial/graphical.rst

```diff
@@ -338,5 +338,66 @@ This adds the input line above the Queue and Abort buttons.
 
 A completer is implemented allowing to quickly select an existing folder, and a button on the right side of the input widget opens a browse dialog.
 
+Using the estimator widget
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to provide estimates of the measurement procedure, an EstimatorWidget is provided that allows the user to define and calculate estimates.
+The widget is automatically activated when the :code:`get_estimates` method is added in the :code:`Procedure`.
+
+The quickest and most simple implementation of the :code:`get_estimates` function simple returns a the estimated duration of the measurement in seconds (as an :code:`int` or a :code:`float`).
+As an example, in the example provided in the ManagedWindow section, the :code:`Procedure` is changed to:
+.. code-block:: python
```

Comment:
An empty line is needed to have the code-block statement understood

Reply:
Indeed, thanks